public/services/SavedObjectService.ts

```diff
@@ -39,10 +39,17 @@ export default class SavedObjectService implements ISavedObjectsService {
               },
             };
           });
-          const createAliasRes = await this.indexService.updateAliases({
+          await this.indexService.updateAliases({
             actions: indexActions,
           });
 
+          const detectorReferences = [
+            {
+              id: detectorId,
+              name: name,
+              type: 'detector-SA',
```

Comment:
Could you clarify the different `type`s that might be available? Would it be worthwhile to create a constant to store an exhaustive list of `type`s?

Reply:
Good question Thomas. Not really sure as this was a part of refactored code. I will check with Amardeep.

Reply:
There is only one type we use for this which is `detector-SA`